content/en/status.md

```diff
@@ -9,42 +9,38 @@ description: Maturity-level of the main OpenTelemetry components
 
 ## {{% param title %}}
 
-OpenTelemetry is made up of [several components][main-comp], some
-language-specific and others language-agnostic. When looking for a status, make
-sure to look for the status from the right component page. For example, the
+OpenTelemetry is made up of [several components](/docs/concepts/components/),
+some language-specific and others language-agnostic. When looking for a status,
```

Comment:
Could we add something here about what "Status" is meant to mean or represent? Or link to the relevant verbiage.

Reply:
```suggestion
some language-specific and others language-agnostic. When looking for a [status](/docs/specs/otel/versioning-and-stability/),
```

like this?

Reply:
That'll do!